extensions/applicationinsights-analytics-js/Tests/ApplicationInsights.tests.ts

```diff
@@ -293,7 +293,7 @@ export class ApplicationInsightsTests extends TestClass {
                     [plugin]
                 );
                 var appInsights = new ApplicationInsights();
-                appInsights.initialize({instrumentationKey: core.config.instrumentationKey}, core, []);
+                appInsights.initialize({instrumentationKey: core.baseCore.config.instrumentationKey}, core, []);
```

Comment:
config must be the same between core and baseCore, please test this using core.config.instrumentationKey